### PR TITLE
Added goerli and kovan networks

### PR DESCRIPTION
**Pull request by kurabi1966**
Added goreli and kovan networks.

[https://docs.api3.org/airnode/v0.2/reference/airnode-addresses.html](url)

### CODE REVIEW

File: packages/airnode-node/src/evm/networks.ts [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import { ethers } from 'ethers';
+// Documentation link: https://docs.api3.org/airnode/v0.2/reference/airnode-addresses.html
```
Comment (bbenligiray) [inline packages/airnode-node/src/evm/networks.ts:2]:
```suggestion
// Documentation link: https://docs.api3.org/airnode/latest/reference/airnode-addresses.html
```

**Review by Siegrift — APPROVED** [review]
:+1: Thanks for this.

LGTM.

File: packages/airnode-node/src/evm/networks.ts [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
 /*
  * Networks:
  *
  * 1 - Ethereum Mainnet
  * 3 - Ethereum Ropsten
  * 4 - Ethereum Rinkeby
+ * 5 - Ethereum goerli
```
Comment (andreogle) [inline packages/airnode-node/src/evm/networks.ts:10]: Just for consistency with the other networks
```suggestion
 * 5 - Ethereum Goerli
```

File: packages/airnode-node/src/evm/networks.ts (hunk 1 above)
Comment (Siegrift) [reply packages/airnode-node/src/evm/networks.ts:2]: We also need to tackle this on the global scale - there are many references to 0.2 and 0.3

File: packages/airnode-node/src/evm/networks.ts [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+ * 42 - Ethereum kovan
```
Comment (andreogle) [inline packages/airnode-node/src/evm/networks.ts:11]:
```suggestion
 * 42 - Ethereum Kovan
```

**Review by andreogle — APPROVED** [review]
Nice one on your first Airnode contribution 🎉 

Don't forget to add a changeset file as well by running `yarn changeset`

File: packages/airnode-node/src/evm/networks.ts (hunk 1 above)
Comment (bbenligiray) [reply packages/airnode-node/src/evm/networks.ts:2]: I'll create an issue

**Review by Siegrift — APPROVED** [review]

**Comment by Siegrift** [comment]
Feel free to merge. I don't think changeset is strictly necessary for this.

**Pull request merged**